here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/NakshaVersion.java

```diff
@@ -50,6 +50,7 @@ public class NakshaVersion implements Comparable<NakshaVersion> {
   public static final String v2_0_13 = "2.0.13";
   public static final String v2_0_14 = "2.0.14";
   public static final String v2_0_15 = "2.0.15";
+  public static final String v2_0_16 = "2.0.16";
```

Comment:
I missed mentioning, but we also should update version in gradle.properties and openapi.yml